use-timescale/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -31,84 +31,91 @@ For more information, see the documentation about
 
 ## Continuous aggregates with a `JOIN` clause
 
-In TimescaleDB 2.10.0 and later, continuous aggregates support JOINS, as long as
-they meet these conditions:
+Continuous aggregates supports the following JOINS.  
+
+| Feature | PostgreSQL | TimescaleDB | 
```

Comment:
```suggestion
| Feature | TimescaleDB < 2.10.x | TimescaleDB <= 2.15.x | TimescaleDB >= 2.16.x| 
```
This is what I was thinking about... because we'll describe a kind of JOIN supported by TSDB version ranges... WDYT?

Reply:
Perfect. Do you have a list for me so I can add it in?